packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -375,6 +375,7 @@ const runAPI = async (plugin, api, args, activity) => {
     const apiCallArgs = [
       {
         ...args,
+        parentSpan: pluginSpan,
```

Comment:
plugin spans were being set as children of the `run-api` span instead of the `run-plugin` span — this fixes that